pkg/scannerv4/client/client.go

```diff
@@ -100,7 +101,12 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 		connOpt.TLS.UseClientCert = clientconn.MustUseClientCert
 		connOpt.TLS.ServerName = o.serverName
 	}
-	return clientconn.GRPCConnection(ctx, o.mTLSSubject, o.address, connOpt)
+
+	callOpts := []grpc.CallOption{
+		grpc.MaxCallRecvMsgSize(env.ScannerV4MaxRespMsgSize.IntegerSetting()),
```

Comment:
What about <https://github.com/stackrox/stackrox/blob/jvdm%2Ffix-dep-nvdtools/scanner/indexer/remote.go#L27>?

Reply:
Or maybe are you planning to onboard `pkg/scannerv4/client` there?

Reply:
good catch going to update `remote.go`. I thought about onboarding the client, but it was proving to be more effort than it was worth, but we could always consider it in the future